html-forms/index.html

```diff
@@ -0,0 +1,47 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <title>HTML Forms</title>
+</head>
+<body>
+  <h1>Coding Form!</h1>
+  <form action="index.html" method="get">
+    <label for="email">Email</label>
+    <div>
+      <input id="email" type="text" name="email" maxlength="50" />
+    </div>
+    <label for="coolcoding">Why is coding cool?</label>
+    <div>
+      <select id="coolcoding" name="coolcodingoption">
+        <option value="Select an option">Select an option</option>
+        <option value="Because it's interesting">Because it's interesting</option>
```

Comment:
small nitpick. Instead of `'` please use HTML entity `&apos;` on line 17, 18, 19

Reply:
noted!